pkg/setting/setting.go

```diff
@@ -518,6 +521,10 @@ func NewConfigContext(args *CommandLineArgs) error {
 	SnapShotRemoveExpired = snapshots.Key("snapshot_remove_expired").MustBool(true)
 	SnapShotTTLDays = snapshots.Key("snapshot_TTL_days").MustInt(90)
 
+	// read dashboard settings
+	dashboards := Cfg.Section("dashboards")
```

Comment:
I think that is a better section name, update defaults.ini and sample.ini to match that section name (it is [dashbards.history] above so right now they do not match).

Reply:
Update `[dashboards.history]` to `[dashboards]`, right?

Reply:
yes

Reply:
ok